### ISSUE TITLE
Plugin keyboard shorcut keys do not work in Obsidian notes after reviewing flahscards [BUG]

### ISSUE THREAD

**Issue by Ivix42** [st3v3nmw/obsidian-spaced-repetition#907]
**Describe the bug**
After reviewing daily flashcards, the plugin designated shortcut keys, "1", "2", "3", "s", and "space" do not function in obsidian without restarting the app. No issues with keys in other apps.

**To Reproduce**

1. Go to Review Flashcards
2. Review all flashcards
3. Return to any other obsidian note.

**Expected behavior**
Keys should still work in obsidian after reviewing flashcards.

**Screenshots**
If applicable, add screenshots to help explain the bug. This could be the logs in the developer console (`CTRL` + `SHIFT` + `i`).

**Versions (please complete the following information):**

-   OS: Win 11
-   Obsidian version: 1.5.11
-   Plugin version: 1.12.0
-   If on desktop, Installer version: [run `this.navigator.appVersion` on the developer console, e.g. `5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) obsidian/0.12.3 Chrome/89.0.4389.128 Electron/12.0.6 Safari/537.36"`]

**Additional context**
Add any other context about the problem here (e.g. the markdown producing the error).

**Comment by Xeretis** [comment]
I'm also experiencing this

**Comment by ronzulu** [comment]
Hey @KyleKlus, this might be your area of expertise, would you have a chance to look into this?

Cheers
Ronny

**Comment by KyleKlus** [comment]
Yes, I found the problem and I'll try to fix it as quick as possible, as this is quite a bad one, if it locks the keys 🙄

**Comment by KyleKlus** [comment]
Alright, I noticed, I already fixed the bug by accident in my UI overhaul in PR #899. As it is already approved by st3v3nmw and merge ready, it would be just easier to wait until it'll be merged.

**Comment by st3v3nmw** [comment]
Merged #899, please try it out.

**Comment by Ivix42** [comment]
The "s" and "space" key are still locked. The 1-3 keys work a few times and then locked as well.

**Comment by KyleKlus** [comment]
Alright, then I need to look deeper into what causes this, as I thought that it was fixed by removing the key listener after it isn't needed.

I may also want to look into how other plugins do the key listeners, so that we could even make the keys reconfigureable in the hotkeys section.

**Comment by KyleKlus** [comment]
I am very sorry, but I can't reproduce the bug. Everything works fine for me. I could maybe reimplement the hotkeys in another way and see if it changes things.

**Comment by Ivix42** [comment]
Just double checked after troubleshooting other plugins. Appears to be a good fix now. Thanks for the work on it!
